utils/connect.js

```diff
@@ -0,0 +1,60 @@
+const { Sequelize, DataTypes, Model } = require('sequelize');
```

Comment:
suggestion: doesn't feel like these are utils - id suggest putting the sequelize stuff in a `lib/db.js` file